client/src/Clusters/TopCharts/SingleAppContainer.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+ 
+class SingleAppContainer extends Component {
+  render() {
+    return (
+      <div>
+        <p class="num"> {this.props.listNum} </p>
+        <img
+          id="app_icon_img"
+          class="img-fluid"
+          alt="icon 1"
+          src={this.props.url}
+        />
+        <div class="app_details">
+          <h6 class="app_title"> {this.props.title} </h6>
+          <p class="grey_text"> {this.props.category} - Category </p>
+          <p class="grey_text">
+            {this.props.rating}
+            <img
+              id="rating_star_icon"
+              width="12px"
```

Comment:
move styling to css

(note: when using inline width and height attributes, you should omit the "px", as it is already implied. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Attributes)